homeassistant/components/dsmr/manifest.json

```diff
@@ -7,5 +7,5 @@
   "integration_type": "hub",
   "iot_class": "local_push",
   "loggers": ["dsmr_parser"],
-  "requirements": ["dsmr_parser==0.33"]
+  "requirements": ["dsmr_parser==1.1.0"]
```

Comment:
As 1.2.0 was just released, would it be possible to bump the version here?

Reply:
Well its not needed and might break things. As there were quite a lot of changes in 1.2.0 for the mbus devices.
But there are still things that need to be fixed in 1.2.0 to have it like we want.
Once that is the case, I think a new patch for HA should be created.
For now, it works fine with 1.1.0